events/yearly_events/yearly_events_5.txt

```diff
@@ -6796,6 +6805,11 @@ yearly.9020 = {
 	
 	option = { #Release the Elephant
 		name = {
+			trigger = { like_hinduism_religion_trigger = yes }
```

Comment:
` yearly.9020.d1:1 "Let us honor [ROOT.Char.GetFaith.KnowledgeGodName] and release it."`
Will sounds weird, but not really much.

Reply:
Commented in 1b4c671f2df31fc8c76a8b92ef76cddc6b9da116.